gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -11,29 +11,54 @@
         "description": "List of limiters to be applied",
         "type": "array",
         "items": {
-          "oneOf": [{
+          "anyOf": [{
             "type": "object",
             "properties": {
               "name": {
                 "type": "string",
                 "enum": ["connections"],
-                "description": "limiting request concurrency (or concurrent connections)"
+                "description": "Limiting request concurrency (or concurrent connections)"
               },
               "key": {
-                "description": "Key of limiter",
-                "type": "string"
+                "description": "The key corresponding to the limiter object",
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string",
+                    "description": "The name of the key, must be unique in the scope"
+                  },
+                  "scope": {
```

Comment:
I think we could extract this into shared definition in this schema an reuse it so it is not duplicated 3 times. 

See an example: 
https://github.com/3scale/apicast/blob/4ab2ac80d1bd6f952de4a0068932b5a992619e0b/gateway/src/apicast/policy/manifest-schema.json#L11-L21

Reply:
Possibly even the whole `key` property could be shared between all limiters.

Reply:
@y-tabata it should be:

```json
{ "properties": {
"key": {
      "$ref": "#/definitions/key"
    }
}
```

You have `"$ref"` as a property of `"properties"`, not `"key" property`.

Reply:
@mikz Thank you!